csharp/src/Google.Protobuf/MessageParser.cs

```diff
@@ -379,7 +379,16 @@ public new T ParseFrom(CodedInputStream input)
         public new T ParseJson(string json)
```

Comment:
Does the other `ParseJson` method on line 179 need this same fix?

Reply:
It looks like it does. Will wait for the outcome of the discussion and then fix as required.